### ISSUE TITLE
New users are synced with DCSOps without a corresponding RCO ID which denies them the ability to login

### ISSUE THREAD

**Issue by OhMcGoo** [redcross/arcdata#100]
Users receive an error message stating that their credentials are invalid. Currently, all known issues are being resolved manually:

- User’s RCO ID is identified from Region’s Volunteer Connection Active Members report
- User is found in Admin > Roster > People
- Edit link is clicked to open Edit User page
- RCO ID is entered into RCO field and the Update Person button is clicked

**Comment by OhMcGoo** [comment]
Pull request #104 attempts to address this issue.

**Comment by kfogel** [comment]
Have we verified that this issue is fixed and can be closed now, @bnazari and @OhMcGoo ?  (CC'ing @pjsier and @pmacaluso3 just so they're in the loop.)

**Comment by OhMcGoo** [comment]
No, this remains unresolved. Yesterday, for example, I received a request from Greater New York Region Volunteer Philip Koehler for help accessing DCSOps. Neither his Volunteer Connection positions nor his RCO ID had transferred to the application
<img width="1036" alt="philip koehler" src="https://user-images.githubusercontent.com/4147733/33232870-32ec24a6-d1d3-11e7-95a5-ce1b26c7b4ad.png">
.

**Comment by bnazari** [comment]
Interesting. I know about RCO id, that still has to be finished, but I need to look into this one a bit. I’ll report back shortly.

**Comment by bnazari** [comment]
Upon review: Volunteer Connection has the start date for the positions set for the 25th of November (human error). It checks start/end dates for positions and qualifications. I suspect this has something to do with it, especially given that Jim was emailed about this yesterday. 

```
    if check_dates
      return unless position_end.nil? or parse_time(position_end) > Time.now
      return unless position_start.nil? or parse_time(position_start) < Time.now
    end
```

As far as RCO ID goes, I'm still working on that part. Will have an answer by tomorrow.

**Comment by bnazari** [comment]
This should resolve the RCO_ID issue. 
#124

**Comment by cecilia-donnelly** [comment]
@bnazari, @OhMcGoo, have we heard from users about whether #124 fixed this?

**Comment by kfogel** [comment]
@bnazari confirms that this is now completely resolved by the deployment of #124.  Closing.
